src/main/java/com/playmonumenta/structures/commands/ListRespawningStructures.java

```diff
@@ -13,18 +16,13 @@ public static void register() {
 		final String command = "listrespawningstructures";
 		final CommandPermission perms = CommandPermission.fromString("monumenta.structures");
 
-		new CommandAPICommand(command)
-			.withPermission(perms)
-			.executes((sender, args) -> {
-				list(sender, null);
-			})
-			.register();
+		Argument<String> labelArg = new StringArgument("label").replaceSuggestions(RespawnManager.SUGGESTIONS_STRUCTURES);
 
 		new CommandAPICommand(command)
 			.withPermission(perms)
-			.withArguments(new StringArgument("label").replaceSuggestions(RespawnManager.SUGGESTIONS_STRUCTURES))
+			.withOptionalArguments(labelArg)
```

Comment:
Same here, if we're checking null later, we shouldn't leave it optional

Reply:
This one should be optional - if it is null it lists all structures